sklearn/utils/extmath.py

```diff
@@ -599,8 +599,10 @@ def log_logistic(X, out=None):
     http://fa.bianp.net/blog/2013/numerical-optimizers-for-logistic-regression/
     """
     is_1d = X.ndim == 1
+    X = np.atleast_2d(X)
```

Comment:
The docstring doesn't conform with the usage of the function. It is passed 1d and that is explicitly handled with the boolean and the ravel.